lib/susedistribution.pm

```diff
@@ -511,9 +511,16 @@ sub activate_console {
             # case the system is still booting (https://bugzilla.novell.com/show_bug.cgi?id=895602)
             # or when using remote consoles which can take some seconds, e.g.
             # just after ssh login
-            assert_screen \@tags, 60;
+            check_screen \@tags, 60;
```

Comment:
Shouldn't we `die` in `else` branch below since we replace `assert_screen` with `check_screen`?

Reply:
Good point, it will be more clear. Thanks!

Reply:
Please put a comment above why we have this standalone `check_screen` which you know I consider a bit tricky and even more so why we do not check the return value. I guess a reference to the progress ticket here as well could be helpful because in the end we want to change the assert_screen back to check_screen ?

Reply:
Actually with current implementation we can use assert_screen here. Then we also can get rid of else branch as we have all tags in there.